src/main/java/agentcommissions/AgentCommissionProcessPage.java

```diff
@@ -369,10 +369,7 @@ public void clickAgentCommissionProcess() throws IOException{
     	AgentCommissionProcessPage sp=new AgentCommissionProcessPage(driver);
         log.info("Click on Agent Commission Process Link in left menu bar");
         WaitClass.sleep(10000);
-        WaitClass.WaitForElementisDisplay(driver, 10, clickAgentCommissionProcess);
-        Assert.assertTrue(clickAgentCommissionProcess.isDisplayed());
-        clickAgentCommissionProcess.click();
-        
+        driver.navigate().to("http://localhost:8080/jbilling/config/partnerCommission");
```

Comment:
@manishagupta592 Please remove portion `http://localhost:8080/jbilling', this part is fixed and it should come from configuration file.